prod-maven-plugin/src/main/java/org/l2x6/cq/maven/prod/ProdInitMojo.java

```diff
@@ -231,10 +231,15 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     final String xPath = PomTunerUtils.anyNs("project", "modules", "module") + "[text() = 'docs']";
                     context.removeNode(xPath, true, true, false);
 
-                    /* Add product module */
-                    getLog().info("Adding to pom.xml: product module");
                     final ContainerElement profileParent = context.getOrAddProfileParent(null);
                     final ContainerElement modules = profileParent.getOrAddChildContainerElement("modules");
+
+                    /* Add integration-tests/message module */
+                    getLog().info("Adding to pom.xml: integration-tests/messaging module");
+                    modules.addChildTextElement("module", "integration-tests/messaging");
```

Comment:
I wonder why we need to add `<module>integration-tests/messaging</messaging>` to the root pom.xml?

Reply:
Well, it looks like a work around.
The `org.apache.camel.quarkus:camel-quarkus-integration-test-messaging-jms` should be set with `${camel-quarkus-community.version}` but this does not happen when running with `cq:prod-excludes`.

If we find a fix, it should be removed.

Reply:
OK, I think I find a cause. Let's take a look at `og.apache.camel.quarkus:camel-quarkus-integration-test-jms-artemis-client` dependencies filtered by `camel-quarkus` extensions:
```
[INFO] contains org.apache.camel.quarkus:camel-quarkus-core
[INFO] contains org.apache.camel.quarkus:camel-quarkus-jms
[INFO] contains org.apache.camel.quarkus:camel-quarkus-support-spring
[INFO] contains org.apache.camel.quarkus:camel-quarkus-bean
[INFO] contains org.apache.camel.quarkus:camel-quarkus-jta
[INFO] contains org.apache.camel.quarkus:camel-quarkus-mock
[INFO] contains org.apache.camel.quarkus:camel-quarkus-xml-jaxp
[INFO] contains org.apache.camel.quarkus:camel-quarkus-support-commons-logging
[INFO] contains org.apache.camel.quarkus:camel-quarkus-direct
```

Then all of them haven been productized. We introduce `camel-quarkus-xml-jaxp` in `3.2.0` recently. So the `jms-artemis-client` test is taken into `TestCategory.PURE_PRODUCT`. Now we have to product `org.apache.camel.quarkus:camel-quarkus-integration-test-messaging-jms` as a transitive dependency.

That could be the reason it keeps `${camel-quarkus.version}` in the `poms/bom-test/pom.xml`.

Reply:
If it really solves some problem, it is fine to leave it there. Anyway it would be nice to add a comment explaining the motivation.

Reply:
Done.